packages/core/src/modules/proofs/protocol/v1/handlers/V1ProposePresentationHandler.ts

```diff
@@ -0,0 +1,103 @@
+import type { V1ProofService } from '..'
+import type { AgentConfig } from '../../../../../agent/AgentConfig'
+import type { Handler, HandlerInboundMessage } from '../../../../../agent/Handler'
```

Comment:
@TimoGlastra I keep noticing it more and more but these `../../../../../../../` are getting a bit much. Can we add something like `@modules`, `@utils` or even `@credentials`? I can create an issue if this is something we would like to add. It would help a lot with readability.